ui/src/app/pages/workspace/workspace-edit.spec.tsx

```diff
@@ -289,30 +290,31 @@ describe('WorkspaceEdit', () => {
     const wrapper = component();
     await waitOneTickAndUpdate(wrapper);
 
-    const accessTierSelection = wrapper.find('[data-test-id="select-access-tier"]');
+    const accessTierSelection = wrapper.find('[data-test-id="select-access-tier"]').find('select');
     expect(accessTierSelection.exists()).toBeTruthy();
 
     // defaults to registered
-    const selectionProps = accessTierSelection.find('select').props();
+    const selectionProps = accessTierSelection.props();
     expect(selectionProps.disabled).toBeFalsy();
     expect(selectionProps.value).toBe(AccessTierShortNames.Registered);
 
     // when Registered is selected, the CDR Version dropdown lists the registered tier CDR Versions
     // defaultCdrVersion and altCdrVersion, with defaultCdrVersion selected
 
-    const cdrVersionsSelect = wrapper.find('[data-test-id="select-cdr-version"]').find('select');
-    expect(cdrVersionsSelect.props().value).toBe(defaultCdrVersion.cdrVersionId);
+    const cdrVersionsSelect = () => wrapper.find('[data-test-id="select-cdr-version"]').find('select');
```

Comment:
I wasn't sure whether checking this again would reference stale data - but this approach makes a refresh  easy and clear.  Thanks.